stdlib/2and3/shutil.pyi

```diff
@@ -96,12 +96,8 @@ else:
                                         Iterable[AnyStr]]] = ...) -> _PathReturn: ...
 
 if sys.version_info >= (3,):
-    @overload
-    def rmtree(path: bytes, ignore_errors: bool = ...,
-               onerror: Optional[Callable[[Any, str, Any], Any]] = ...) -> None: ...
-    @overload
-    def rmtree(path: _AnyPath, ignore_errors: bool = ...,
-               onerror: Optional[Callable[[Any, _AnyPath, Any], Any]] = ...) -> None: ...
+    def rmtree(path: Union[str, bytes, os.PathLike[str]], ignore_errors: bool = ...,
```

Comment:
`Path` can be used instead of `str` and `os.PathLike[str]` to fix CI. This alias works with Python < 3.6.